lesson_02/routes/app.route.js

```diff
@@ -0,0 +1,63 @@
+import { Router } from 'express';
+import { addUserSchema, updateUserSchema, removeUserSchema } from '../schemas/user.schema';
+import { getLoginMatch, handleError } from '../utils';
+
+export const router = Router();
+
+const store = {};
+
+router.get('/list', (req, res) =>  {
```

Comment:
according to the task here should be getting user by id endpoint

Reply:
also, you didn't add `getAutoSuggestUsers` endpoint

Reply:
Done